core/src/com/cloud/agent/resource/virtualnetwork/VirtualRoutingResource.java

```diff
@@ -374,9 +375,9 @@ private Answer execute(AggregationControlCommand cmd) {
                 FileConfigItem fileConfigItem = new FileConfigItem(VRScripts.CONFIG_CACHE_LOCATION, cfgFileName, sb.toString());
                 ScriptConfigItem scriptConfigItem = new ScriptConfigItem(VRScripts.VR_CFG, "-c " + VRScripts.CONFIG_CACHE_LOCATION + cfgFileName);
                 // 120s is the minimal timeout
-                int timeout = answerCounts * _eachTimeout;
-                if (timeout < 120) {
-                    timeout = 120;
+                Duration timeout = _eachTimeout.withDurationAdded(_eachTimeout.getStandardSeconds(), answerCounts);
```

Comment:
Why use `withDurationAdded` rather `multipliedBy`?  Passing `_eachTimeout` into the method is an indication that `multipliedBy` may be a more concise expression of intent.